SU2_CFD/include/drivers/CDriver.hpp

```diff
@@ -130,73 +130,83 @@ class CDriver {
    */  
   virtual void Run() { };
 
+  /*!
+   * \brief Init_Containers
+   */
+  void SetContainers_Null();
+  
   /*!
    * \brief Read in the config and mesh files.
    */
-  void Input_Preprocessing(SU2_Comm MPICommunicator);
+  void Input_Preprocessing(CConfig **&config, CConfig *&driver_config);
```

Comment:
Reference to pointer gives me the creeps, but it does read better.
Please make the preprocessing methods private.

Reply:
Yeah I am also not a very big fan. If we find a better way of indicating what will be allocated, I am open for that too.

Reply:
Still the methods can be private or protected to avoid accidents right?